src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -247,8 +260,14 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       page: page + 1,
       fields: ['id'],
       type: filteredTypes,
-      namespaces: visibleNamespaces,
     };
+
+    const availableNamespaces = namespaceRegistry.getAll()?.map((ns) => ns.id) || [];
+    if (availableNamespaces?.length) {
```

Comment:
Based on line 265, `availableNamespaces` is always an array:
```suggestion
    if (availableNamespaces.length) {
```